qiskit/circuit/instructionset.py

```diff
@@ -55,10 +55,10 @@ def inverse(self):
             self.instructions[index] = instruction.inverse()
         return self
 
-    def q_if(self, *qregs):
+    def control(self, *qregs):
```

Comment:
This is not implemented yet and should probably be removed since currently `control` only optionally takes `num_ctrl_qubits` and `label`.